vertx-web/src/main/java/io/vertx/ext/web/handler/impl/StaticHandlerImpl.java

```diff
@@ -562,7 +571,25 @@ public StaticHandler setAlwaysAsyncFS(boolean alwaysAsyncFS) {
 
   @Override
   public StaticHandler setHttp2PushMapping(List<Http2PushMapping> http2PushMap) {
-    if(http2PushMap != null) this.http2PushMappings = new ArrayList<>(http2PushMap);
+    if(http2PushMap != null) {
+      this.http2PushMappings = new ArrayList<>(http2PushMap);
+    }
+    return this;
+  }
+
+  @Override
+  public StaticHandler skipCompressionForMediaTypes(Set<String> mediaTypes) {
+    if(mediaTypes != null) {
```

Comment:
Have you applied the formatting rules to your IDE? There should be a space between `if` and the opening parenthesis.